public/img/js/controllers/game.js

```diff
@@ -0,0 +1,184 @@
+angular.module('mean.system')
+.controller('GameController', ['$scope', 'game', '$timeout', '$location', 'MakeAWishFactsService', '$dialog', function ($scope, game, $timeout, $location, MakeAWishFactsService, $dialog) {
+    $scope.hasPickedCards = false;
+    $scope.winningCardPicked = false;
+    $scope.showTable = false;
+    $scope.modalShown = false;
+    $scope.game = game;
+    $scope.pickedCards = [];
+    var makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
+    $scope.makeAWishFact = makeAWishFacts.pop();
+
+    $scope.pickCard = function(card) {
+      if (!$scope.hasPickedCards) {
+        if ($scope.pickedCards.indexOf(card.id) < 0) {
+          $scope.pickedCards.push(card.id);
+          if (game.curQuestion.numAnswers === 1) {
+            $scope.sendPickedCards();
+            $scope.hasPickedCards = true;
+          } else if (game.curQuestion.numAnswers === 2 &&
+            $scope.pickedCards.length === 2) {
+            //delay and send
+            $scope.hasPickedCards = true;
+            $timeout($scope.sendPickedCards, 300);
+          }
+        } else {
+          $scope.pickedCards.pop();
+        }
+      }
+    };
+
+    $scope.pointerCursorStyle = function() {
+      if ($scope.isCzar() && $scope.game.state === 'waiting for czar to decide') {
+        return {'cursor': 'pointer'};
+      } else {
+        return {};
+      }
+    };
+
+    $scope.sendPickedCards = function() {
+      game.pickCards($scope.pickedCards);
+      $scope.showTable = true;
+    };
+
+    $scope.cardIsFirstSelected = function(card) {
+      if (game.curQuestion.numAnswers > 1) {
+        return card === $scope.pickedCards[0];
+      } else {
```

Comment:
Loose the else block, nothing else can run except returning false if the execution path doesn't enter the if block

Reply:
The `return false` remains, just remove the else block

Reply:
I didn't make this change. I just copied the files here since i couldn't find them